wacruit/src/database/models/user.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING
+
+from sqlalchemy import Boolean
+from sqlalchemy import String
+from sqlalchemy.orm import Mapped
+from sqlalchemy.orm import mapped_column
+from sqlalchemy.orm import relationship
+
+from wacruit.src.database.models.base import DeclarativeBase
+from wacruit.src.database.models.base import intpk
+from wacruit.src.database.models.base import str30
+from wacruit.src.database.models.base import str50_default_none
```

Comment:
여러 import 를 comma-seperate 하지 않는 이유는 기존에 정해놓은 컨벤션임?

Reply:
아 이거.. 구글 파이썬 가이드인가 그런 컨벤션이 있다고 하는데
그냥 포매터가 해주는대로 따라가기로 결론냈어
나는 어느쪽이든 상관 없음

Reply:
[PEP-8](https://peps.python.org/pep-0008/#imports) 이라고 하는 가장 널리 사용되는 파이썬 스타일 가이드가 있는데, 여기에 따르면 from 문 없이 직접 모듈을 import 할 때는 너가 한 것처럼 여러 line 으로 끊어서 import 하되, 동일한 패키지에서 여러 모듈을 (또는 동일한 모듈에서 여러 클래스/함수를) import 하는 경우에는 comma로 구분해도 괜찮다고 하거든
그래서 어느쪽을 선택하든 취향 껏 고르면 될 것 같긴한데 난 동일한 패키지/모듈은 반복해서 작성하는 것보다 single line으로 적거나 만약 그게 너무 길다면 괄호로 묶는 것도 좋을 것 같은데 어느 쪽이든 투표해서 통일시키자
```suggestion
from wacruit.src.database.models.base import (
    DeclarativeBase,
    intpk,
str30,
str50_default_none
```

Reply:
전찬성~

Reply:
지금 precommit에 적용된 포매터(black)이 이 포맷을 강제하는거같은데,, 이거 바꾸려면 나중에 precommit 설정을 바꾸든지 해야할듯?
이 PR에서는 수정 안하고 있을게